platform/core/src/classes/ImageSet.ts

```diff
@@ -32,6 +34,24 @@ class ImageSet {
     });
   }
 
+  get images(): any {
```

Comment:
The general design of the display sets is that they are fixed representations of specific groups of DICOM objects - this mapping allows redefining that, and can cause problems when rendering different image sets.  I'd rather have a mapping function produce a new ImageSet entirely rather than having the images be variable.

Reply:
It isn't used anywhere here, and I don't see any testing behaviour for it - I think this is a bad idea without careful consideration of what it means for other classes which expect to the representation of display sets to be consistent.